src/dao/LOCsManagerDAO.js

```diff
@@ -2,6 +2,7 @@ import {Map} from 'immutable'
 import AbstractContractDAO from './AbstractContractDAO'
 import LOCDAO, {Setting, SettingString, SettingNumber} from './LOCDAO'
 import LOCNoticeModel, {ADDED, REMOVED, UPDATED} from '../models/notices/LOCNoticeModel'
+import {bytes32} from '../utils/bytes32'
```

Comment:
this _bytes32_ works wrong
whole _utils/bytes32.js_ should be deleted and replaced with _AbstractContractDAO _bytesToString & _toBytes32_ accordingly